sale_partner_incoterm/tests/test_sale_partner_incoterm.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2015 Opener B.V. (<https://opener.am>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestSalePartnerIncoterm(TransactionCase):
+    def test_sale_partner_incoterm(self):
+        """
+        Check that the customer's default incoterm is retrieved in the
+        sales order's onchange
+        """
+        customer = self.env["res.partner"].search([("customer_rank", ">", 1)], limit=1)
+        incoterm = self.env["account.incoterms"].search([], limit=1)
+        customer.write({"sale_incoterm_id": incoterm.id})
+        sale_order = self.env["sale.order"].create({"partner_id": customer.id})
+        sale_order.onchange_partner_id()
```

Comment:
to test the onchanges it's better to use the form class https://github.com/odoo/odoo/blob/13.0/odoo/tests/common.py#L1336
No a blocker, but keep it mind of the next time ;)